code/game/objects/items/weapons/switchtool.dm

```diff
@@ -184,6 +192,7 @@
 
 	icon_state = "s_a_k"
 	desc = "Crafted by the Space Swiss for everyday use in military campaigns. Nonpareil."
+	no_remove = 0
```

Comment:
This is unnecessary, it would inherit the value of `0` for its `no_remove` from the regular switchtool anyway.